Source/Core/Serializer.swift

```diff
@@ -14,3 +14,14 @@ public protocol Serializer {
     
     func deserialize(_ data: Data) -> SupportedType
 }
+
+extension Serializer {
+    
+    public func serializeToString(_ supportedType: SupportedType) -> String {
+        return String(data: serialize(supportedType), encoding: .utf8) ?? ""
+    }
+    
+    public func deserializeFromString(_ string: String) -> SupportedType {
```

Comment:
What's the correct naming here,
`deserializeFromString(_ string: String)` or `deserialize(fromString string: String)` or `deserialize(from string: String)`.

Reply:
The last one seems most fitting for Swift to me.